### DISCUSSION TITLE
7.6.1 - issue: AttributeError

### DISCUSSION THREAD

**Discussion by zolo-o** [UltimaHoarder/UltimaScraper#1934] [Q&A] [answered]
Version: 7.6.1
OS: Ubuntu 22.04.1 LTS
$ python3 -V
Python 3.10.6

$ pip3 -V
pip 22.2.2 from /usr/local/lib/python3.10/dist-packages/pip (python 3.10)

ERROR: 
---
Scrape Processing
Name: sxxxxxm
Scrape: 0 = All | 1 = Images | 2 = Videos | 3 = Audios | 4 = Texts
0
Type: Profile
Type: Stories
Traceback (most recent call last):
  File "/home/zzzz/of/OnlyFans-7.6.1/start_ofd.py", line 104, in <module>
    asyncio.run(main())
  File "/usr/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/home/zzzz/of/OnlyFans-7.6.1/start_ofd.py", line 86, in main
    api = await main_datascraper.start_datascraper(
  File "/home/zzzz/of/OnlyFans-7.6.1/datascraper/main_datascraper.py", line 94, in start_datascraper
    await main_helper.process_names(
  File "/home/zzzz/of/OnlyFans-7.6.1/helpers/main_helper.py", line 966, in process_names
    result = await module.start_datascraper(authed, username, site_name)
  File "/home/zzzz/of/OnlyFans-7.6.1/modules/onlyfans.py", line 161, in start_datascraper
    await prepare_scraper(authed, site_name, item)
  File "/home/zzzz/of/OnlyFans-7.6.1/modules/onlyfans.py", line 784, in prepare_scraper
    highlights = await subscription.get_highlights()
  File "/home/zzzz/of/OnlyFans-7.6.1/apis/onlyfans/classes/create_user.py", line 270, in get_highlights
    results = [create_highlight(x) for x in results]
  File "/home/zzzz/of/OnlyFans-7.6.1/apis/onlyfans/classes/create_user.py", line 270, in <listcomp>
    results = [create_highlight(x) for x in results]
  File "/home/zzzz/of/OnlyFans-7.6.1/apis/onlyfans/classes/create_highlight.py", line 3, in __init__
    self.id: int = option.get("id")
AttributeError: 'str' object has no attribute 'get'

---
pip list (under user account)
`Package                 Version
----------------------  ---------------
aiohttp                 3.8.3
aiohttp-socks           0.7.1
aiosignal               1.2.0
alembic                 1.8.1
apturl                  0.5.2
async-generator         1.10
async-timeout           4.0.2
attrs                   22.1.0
bcrypt                  3.2.0
beautifulsoup4          4.11.1
blinker                 1.4
Brlapi                  0.8.3
certifi                 2022.9.24
chardet                 4.0.0
charset-normalizer      2.1.1
click                   8.0.3
colorama                0.4.4
command-not-found       0.3
commonmark              0.9.1
cryptography            3.4.8
cupshelpers             1.0
dbus-python             1.2.18
defer                   1.0.6
distro                  1.7.0
distro-info             1.1build1
duplicity               0.8.21
exceptiongroup          1.0.0rc9
fasteners               0.14.1
frozenlist              1.3.1
future                  0.18.2
greenlet                1.1.3
h11                     0.14.0
httplib2                0.20.2
idna                    3.4
importlib-metadata      4.6.4
jeepney                 0.7.1
jsonpickle              2.2.0
keyring                 23.5.0
language-selector       0.1
launchpadlib            1.10.16
lazr.restfulclient      0.14.4
lazr.uri                1.0.6
lockfile                0.12.2
louis                   3.20.0
lxml                    4.9.1
macaroonbakery          1.3.1
Mako                    1.2.3
MarkupSafe              2.1.1
mergedeep               1.3.4
monotonic               1.6
more-itertools          8.10.0
multidict               6.0.2
netifaces               0.11.0
oauthlib                3.2.0
olefile                 0.46
outcome                 1.2.0
paramiko                2.9.3
pexpect                 4.8.0
Pillow                  9.0.1
pip                     22.2.2
protobuf                3.12.4
psutil                  5.9.2
ptyprocess              0.7.0
pycairo                 1.20.1
pycups                  2.0.1
Pygments                2.13.0
PyGObject               3.42.1
PyJWT                   2.3.0
pymacaroons             0.13.0
PyNaCl                  1.5.0
pyparsing               2.4.7
pyRFC3339               1.1
PySocks                 1.7.1
python-apt              2.3.0+ubuntu2.1
python-dateutil         2.8.2
python-debian           0.1.43ubuntu1
python-socks            2.0.3
pytz                    2022.1
pyxdg                   0.27
PyYAML                  5.4.1
reportlab               3.6.8
requests                2.28.1
rich                    12.6.0
SecretStorage           3.3.1
selenium                4.5.0
selenium-wire           2.1.2
setuptools              65.4.1
six                     1.16.0
sniffio                 1.3.0
sortedcontainers        2.4.0
soupsieve               2.3.2.post1
SQLAlchemy              1.4.20
ssh-import-id           5.11
systemd-python          234
tqdm                    4.64.1
trio                    0.22.0
trio-websocket          0.9.2
ubuntu-advantage-tools  27.10
ubuntu-drivers-common   0.0.0
ufw                     0.36.1
ujson                   5.5.0
unattended-upgrades     0.1
urllib3                 1.26.12
usb-creator             0.3.7
user-agent              0.1.10
wadllib                 1.3.6
wheel                   0.37.1
win32-setctime          1.1.0
wsproto                 1.2.0
xdg                     5
xkit                    0.0.0
yarl                    1.8.1
zipp                    1.0.0`

**Comment by zolo-o** [comment, marked as answer]
Closing

**Reply by vnavspd** [reply]
Hi, i'm getting the same error AttributeError: 'str' object has no attribute 'get'.

Is there a fix for this? I'm not really understanding your post marked as the answer.
Thanks

**Reply by zolo-o** [reply]
Actually I've closed this as I've mixed up two repos. This is the "Digitalcriminal" and latest version is 7.4.1. The "Digitalcriminals" latest version is 7.6.1. Nevertheless none of them working. But thought no reason for this question since it should be asked at the other github profile. No idea yet how this will get fixed. Would be happy with at least one working version.
